Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -679,6 +679,20 @@ public struct HTTPVersion: Equatable {
         }
     }
 
+    /// HTTP/3
+    public static let http3 = HTTPVersion(major: 3, minor: 0)
+
+    /// HTTP/2
+    public static let http2 = HTTPVersion(major: 2, minor: 0)
+
+    /// HTTP/1.1
+    public static let http11 = HTTPVersion(major: 1, minor: 1)
```

Comment:
Don't like the name.  Why not `http1_1` or similar?

Reply:
I don't love underscores in the name. Happy to throw it to the team for a tiebreaker: @weissi @glbrntt @Davidde94.

Reply:
Oh no, now I'm over-thinking it. I think I agree with Peter...

Reply:
I don't love either but I prefer the underscores.